mcux/middleware/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-if(CONFIG_BT OR CONFIG_NET_L2_IEEE802154 OR CONFIG_NET_L2_OPENTHREAD)
```

Comment:
`CONFIG_NET_L2_OPENTHREAD` can't be remove.
It's used for redfinch (RW) who has a RCP mode.

^^ @JA-NXP